Vagrantfile~

```diff
@@ -0,0 +1,41 @@
+Vagrant.configure('2') do |config|
```

Comment:
(Your vagrant configuration file should probably not be in the exercism.io repository.)